tests/unittests/utils/test_paddleocr_vl.py

```
@@ -0,0 +1,255 @@
+
+    def test_recognition_task_accepted(self):
+        config = foup.PaddleOCRVLModelConfig({"task": "table"})
+        self.assertEqual(config.task, "table")
+
+
+class PixelBudgetEffectTests(unittest.TestCase):
+    """Verifies the size bounds change the processor's output resolution,
+    not merely that they are forwarded."""
+
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.15.21)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)